spark/dl/src/main/scala/com/intel/analytics/bigdl/package.scala

```diff
@@ -37,4 +39,41 @@ package object bigdl {
   val numeric = com.intel.analytics.bigdl.tensor.TensorNumericMath.TensorNumeric
 
   type DataSet[D] = AbstractDataSet[D, _]
+
+  private object BigDLBuildInfo {
+
+      val (version: String) = {
```

Comment:
I did this as there might be more properties to add in future , surely we can change it to one val for now, makes more sense.